src/mergerepo_c.c

```diff
@@ -164,6 +166,10 @@ static GOptionEntry cmd_entries[] =
       "Show program's version number and exit", NULL },
     { "repo", 'r', 0, G_OPTION_ARG_FILENAME_ARRAY, &(_cmd_options.repos),
       "Repo url", "REPOS" },
+    { "repo-prefix", 0, 0, G_OPTION_ARG_STRING, &(_cmd_options.repo_prefix),
+      "Repository prefix to be replaced by repo-prefix-url." },
+    { "repo-prefix-url", 0, 0, G_OPTION_ARG_STRING, &(_cmd_options.repo_prefix_url),
+      "Repository prefix URL by which the repo-prefix is replaced." },
```

Comment:
Make sure the options must be used together.

python-like meta-code follows :) :
if repo_prefix_search != NULL or repo_prefix_replace != NULL:
   (I assume that NULL means that option hasn't been set)
   if repo_prefix_search == NULL: error
   if repo_prefix_replace == NULL: error
   (they have to be used together)

   for repo in repos:
      search for prefix and replace it
      if repo_prefix_search == "": prepend new prefix to all repos
      if repo_prefix_replace == "": remove prefixes

(I hope the option parser allows this behavior)